tests/Feature/FavoritesControllerTest.php

```diff
@@ -37,4 +37,13 @@ public function testCityCanBeRemovedFromFavorites(): void
             "city_id" => $city->id,
         ]);
     }
+
+    public function testUserIsNotAuthenticated(): void
```

Comment:
I would like to rename test to something like:
`testUnauthenitactedUserCannotAddCityToFavourites`

Reply:
Done